unittests/testpublicevents.py

```diff
@@ -33,35 +49,67 @@ class TestPublicEvents(TestCase):
     maxDiff = None
 
     def setUp(self):
-        self.arch = Archivist("url", None, max_time=1)
+        self.public = ArchivistPublic(max_time=1)
 
     def tearDown(self):
-        self.arch = None
+        self.public.close()
 
     def test_publicevents_str(self):
         """
         Test events str
         """
         self.assertEqual(
-            str(self.arch.publicevents),
-            "PublicEventsClient(url)",
+            str(self.public.events),
+            "EventsPublic()",
             msg="Incorrect str",
         )
 
     def test_publicevents_read(self):
         """
         Test publicevent reading
         """
-        with mock.patch.object(self.arch._session, "get") as mock_get:
+        with mock.patch.object(self.public.session, "get") as mock_get:
             mock_get.return_value = MockResponse(200, **RESPONSE)
 
-            event = self.arch.publicevents.read(IDENTITY)
+            event = self.public.events.read(EVENT_IDENTITY)
             self.assertEqual(
                 tuple(mock_get.call_args),
                 (
                     (
                         (
-                            f"url/{ROOT}/{PUBLICASSETS_SUBPATH}"
+                            f"{URL}/{ROOT}/{PUBLICASSETS_SUBPATH}"
```

Comment:
Again check whether reading with subpath present is supposed to work

Reply:
subsequent PR fixes this...